actiontext/lib/action_text/content.rb

```diff
@@ -1,6 +1,24 @@
 # frozen_string_literal: true
 
 module ActionText
+  # = Action Text \Content
+  #
+  # The ActionText::Content class wraps an HTML fragment to add support for
```

Comment:
Do we have guidelines for whether a class should be linking to itself or not (assuming that `ActionText::Content` here will be a link)? I think I've been seeing it done both ways.

Reply:
Good point. I have no idea 😄 
I'm seeing either:
```suggestion
  # Content wraps an HTML fragment to add support for ...
```
or
```suggestion
  # Wraps an HTML fragment to add support for ...
```

Reply:
Right, I've also seen some which will use the class name but use `<tt>` to not link to it (since a user would already be on that page):
https://github.com/rails/rails/blob/533161a73df4bfaccb7856dad45c531ac6cafde7/activesupport/lib/active_support/concern.rb#L21
https://github.com/rails/rails/blob/533161a73df4bfaccb7856dad45c531ac6cafde7/activesupport/lib/active_support/log_subscriber.rb#L11

Reply:
Thanks. I went with the `<tt>` per: https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#fixed-width-font

Reply:
I think for classes/modules you can also use the `+...+` shortcut, e.g.:

```
$ echo "+Foo::Bar+" | rdoc --pipe

#=> <p><code>Foo::Bar</code></p>
```